contracts/MarketCapSqrtController.sol

```diff
@@ -333,32 +341,59 @@ contract MarketCapSqrtController is MarketCapSortedTokenCategories {
   }
 
   /**
-   * @dev Sets the maximum number of pool tokens that can be minted
-   * for a particular pool.
-   *
-   * This value will be used in the alpha to limit the maximum damage
-   * that can be caused by a catastrophic error. It can be gradually
-   * increased as the pool continues to not be exploited.
-   *
-   * If it is set to 0, the limit will be removed.
-   *
-   * @param poolAddress Address of the pool to set the limit on.
-   * @param maxPoolTokens Maximum LP tokens the pool can mint.
+   * @dev Sets the default exit fee recipient for new pools.
    */
-  function setMaxPoolTokens(
-    address poolAddress,
-    uint256 maxPoolTokens
-  ) external onlyOwner _havePool(poolAddress) {
-    IIndexPool(poolAddress).setMaxPoolTokens(maxPoolTokens);
+  function setDefaultExitFeeRecipient(address defaultExitFeeRecipient_) external onlyOwner {
+    require(defaultExitFeeRecipient_ != address(0), "ERR_NULL_ADDRESS");
+    defaultExitFeeRecipient = defaultExitFeeRecipient_;
   }
 
   /**
-   * @dev Sets the swap fee on an index pool.
+   * @dev Sets the exit fee recipient on an existing pool.
+   */
+  function setExitFeeRecipient(address poolAddress, address exitFeeRecipient) external onlyOwner _havePool(poolAddress) {
+    // No not-null requirement - already in pool function.
+    IIndexPool(poolAddress).setExitFeeRecipient(exitFeeRecipient);
+  }
+
+  /**
+   * @dev Sets the exit fee recipient on multiple existing pools.
+   */
+  function setExitFeeRecipient(address[] calldata poolAddresses, address exitFeeRecipient) external onlyOwner {
+    for (uint256 i = 0; i < poolAddresses.length; i++) {
+      address poolAddress = poolAddresses[i];
+      require(_poolMeta[poolAddress].initialized, "ERR_POOL_NOT_FOUND");
+      // No not-null requirement - already in pool function.
+      IIndexPool(poolAddress).setExitFeeRecipient(exitFeeRecipient);
+    }
+  }
```

Comment:
These are functionally different, is that intentional?

Reply:
Ah, didn't notice the modifier

Reply:
They're functionally the same as the modifier logic is placed in the loop. The purpose of this function is to allow the same fee recipient to be set for many pools, as I do not foresee a scenario where we'd need to set different recipients for each pool.